shapely/_geometry.py

```diff
@@ -69,8 +69,7 @@ def get_type_id(geometry, **kwargs):
     ----------
     geometry : Geometry or array_like
     **kwargs
-        For other keyword-only arguments, see the
-        `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
+        See :ref:`NumPy ufunc docs <ufuncs.kwargs>` for other keyword arguments.
```

Comment:
Personally, for _docstrings_ I am a bit on the fence about using intersphinx... The thing is that you also look at this in plain text when interactively checking the help in a terminal/notebook (and not just the online rendered html docs), and in that case the full link is something you can copy/paste, while the intersphinx label is basically useless.

Reply:
A counterpoint is the first link from [a web search of "NumPy ufunc docs"](https://duckduckgo.com/?q=NumPy+ufunc+docs) is the same URL. Unless extensions are enabled, users may need to copy/paste something into a browser -- either a valid URL or phrase. But otherwise, your point is valid, as I often inspect plain-text docstrings too.

Reply:
Yes, that's true, and this is also not a link that users will often have to check (the usage of one of those extra keyword arguments is far from general usage). So given that, also fine with your change.